components/resizable-iframe/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
We should use our `wp.element` abstraction consistently, even in forks.

https://github.com/WordPress/gutenberg/tree/master/element